docs/cudf/source/user_guide/benchmarks.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>The speedups for consequent runs of UDF's are insane! Is it okay to showcase this in those notebook @shwina?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>